src/stride.rs

```diff
@@ -167,6 +167,20 @@ macro_rules! stride_impl {
             pub fn len(&self) -> usize {
                 ((self.end - self.offset) / self.stride) as usize
             }
+
+            /// Return a reference to the element of a stride at the
+            /// given index, or None if the index is out of bounds.
+            #[inline]
+            pub fn get<'b>(&'b self, i: usize) -> Option<&'b A> {
```

Comment:
The lifetime parameter ('b) here is not needed, it matches what the default would be without it. Some of the methods around also have explicit lifetimes, I guess they are a bit old school.

I will fix this up while merging.